### PR TITLE
A few mappings around villagers

### PR DESCRIPTION
**Pull request by liach**
Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

File: mappings/net/minecraft/client/render/entity/model/VillagerResemblingModel.mapping [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CLASS dfz net/minecraft/client/render/entity/model/VillagerResemblingModel
```
Comment (Gegy) [inline mappings/net/minecraft/client/render/entity/model/VillagerResemblingModel.mapping:1]: This name is fine, just wondering if `VillagerLikeModel` might be simpler

Comment (liach) [reply mappings/net/minecraft/client/render/entity/model/VillagerResemblingModel.mapping:1]: I think "resembling" is more accurate, though "like" is a correct term as well (it is used too often and slightly confusing)

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping [hunk 2]
```diff
@@ -15,6 +15,7 @@ CLASS ava net/minecraft/entity/passive/VillagerEntity
 	FIELD bF levellingUp Z
 	FIELD bG lastCustomer Lavn;
 	FIELD bH buddyGolemId Ljava/util/UUID;
+	FIELD bI oneMinAfterLastGolemCheckTimestamp J
```
Comment (Gegy) [inline mappings/net/minecraft/entity/passive/VillagerEntity.mapping:18]: I guess this is `nextGolemCheckTimestamp`?

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping [hunk 3]
```diff
@@ -23,13 +24,15 @@ CLASS ava net/minecraft/entity/passive/VillagerEntity
...
 	METHOD a isLackingBuddyGolem (J)Z
+		ARG 1 time
 	METHOD a wantsGolem (Lahz;)Z
 	METHOD a initBrain (Laix;)V
 	METHOD a releaseTicketFor (Laor;)V
 	METHOD a talkWithVillager (Lava;J)V
 		ARG 1 villager
 		ARG 2 time
 	METHOD a setVillagerData (Lavb;)V
+		ARG 1 arg1
```
Comment (Gegy) [inline mappings/net/minecraft/entity/passive/VillagerEntity.mapping]: arg1?

File: mappings/net/minecraft/entity/passive/VillagerEntity.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/entity/passive/VillagerEntity.mapping:18]: no. This is a trash field left by proguard; it is only set but never read. every time it is set, it is set to one minute after last golem check time if the field's value is less than that.

Comment (Gegy) [reply mappings/net/minecraft/entity/passive/VillagerEntity.mapping:18]: It's state influences the return here, I don't think it's a trash field. here

Comment (Gegy) [reply mappings/net/minecraft/entity/passive/VillagerEntity.mapping:18]: Hm, I guess this code is wrong. Might be a bug, the condition would be true every tick

**Comment by modmuss50** [comment]
Merging due to update, open a new issue or PR with further comments.

**Pull request merged**